dali/python/nvidia/dali/pipeline.py

```diff
@@ -419,6 +427,12 @@ def release_outputs(self):
                 raise RuntimeError("Pipeline must be built first.")
             return self._pipe.ReleaseOutputs()
 
+    def async_release_outputs(self, stream):
+        with self._check_api_type_scope(types.PipelineAPIType.SCHEDULED) as check:
+            if not self._built:
+                raise RuntimeError("Pipeline must be built first.")
+            return self._pipe.AsyncReleaseOutputs(stream.cuda_stream)
```

Comment:
Here you assume that `stream` is PyTorch stream. Can we just ask about numerical value, so user will provide `stream.cuda_stream` as an argument or whatever any other FW provides?